docs/components/modeler/bpmn/undefined-tasks/undefined-tasks.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: undefined-tasks
+title: "Undefined tasks"
+description: "An undefined task defines a task that is external to the BPM engine."
```

Comment:
:thought_balloon: Just for clarification, we're using the term "BPM engine" here as we're describing a BPMN concept used solely by the BPM side of Zeebe.

In other places of the docs, we use the term "workflow engine" to refer to Zeebe as an engine that uses BPMN, DMN, and possibly other concepts to provide automation.

:thinking: It might be good to add this distinction in the glossary page.

Reply:
Thank you for calling this out! I'll note in the glossary.

Reply:
@koevskinikola @christinaausley Do we want to make a change to this?

💭 Side note: The description was copied from Manual Tasks and adjusted for this task type.

🔧 If we adjust it, then we should do it consistently.

💭 AFAIK, we don't render these [front matter](https://docusaurus.io/docs/sidebar/autogenerated#doc-item-metadata) sections anywhere. Normally they are used by Docusaurus to [dynamically construct the sidebar](https://docusaurus.io/docs/sidebar/autogenerated) and categories. But these docs have a statically defined [sidebar](https://docusaurus.io/docs/sidebar/items#sidebar-item-category).

Reply:
I'll create an issue for this @korthout -- I'll want to skim all of the docs for these references. Can keep as-is for now.